src/Compilers/CSharp/Portable/Utilities/InterceptableLocation.cs

```diff
@@ -40,6 +40,8 @@ private protected InterceptableLocation() { }
 
     public abstract override bool Equals(object? obj);
     public abstract override int GetHashCode();
+
+    public abstract bool Equals(InterceptableLocation other);
```

Comment:
Just occurred to me that this is a public API change. I don't see any reason this would be *problematic*--any more than adding the interface, which was included in the originally approved proposal, would be problematic here. But let's check with @dotnet/roslyn-api-owners to make sure.

Reply:
Alternatively, Equals(InterceptableLocation) could be an explicit interface implementation.

Reply:
> Alternatively, Equals(InterceptableLocation) could be an explicit interface implementation.

Yeah, I could do that as well.

> I don't see any reason this would be problematic

Neither do I. But I'll be waiting for a response from @dotnet/roslyn-api-owners